package.json

```diff
@@ -11,14 +11,21 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "lint": "eslint --ext .js,.jsx ./",
     "test": "react-scripts test --env=jsdom",
+    "test:CI": "CI=true react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
     "storybook": "start-storybook -c src/components/.storybook -p 4000",
-    "build-storybook": "build-storybook"
+    "storybook:build": "build-storybook"
   },
+  "precommit": ["lint", "test:CI"],
```

Comment:
pre-commit package that runs these npm scripts in order, and only allows commits when all of them pass.